internal/drivers/pod/pod.go

```diff
@@ -385,6 +385,8 @@ func (o *opts) startLogStream(ctx context.Context, podName string) <-chan error
 		defer close(errch)
 
 		scanner := bufio.NewScanner(logs)
+		buf := make([]byte, 0, 1024*1024)
+		scanner.Buffer(buf, 1024*1024)
```

Comment:
```suggestion
		buf := make([]byte, 0, 512*1024)
		scanner.Buffer(buf, 512*1024)
```

just a little more conservative given we actually have no idea what we're optimizing for 😓